global.json

```diff
@@ -7,10 +7,10 @@
     "runtimes": {
       "dotnet/x64": [
         "2.1.18",
-        "$(MicrosoftNETCoreAppInternalPackageVersion)"
+        "$(MicrosoftNETCoreBrowserDebugHostTransportPackageVersion)"
       ],
       "dotnet/x86": [
-        "$(MicrosoftNETCoreAppInternalPackageVersion)"
+        "$(MicrosoftNETCoreBrowserDebugHostTransportPackageVersion)"
```

Comment:
Should this be the VS Redist package?

Reply:
We can use the VS Redist package if you'd prefer. This just needs to be any non-shipping package from dotnet/runtime. Since aspnetcore was already consuming this package, I decided to use this one as the sentinel for non-Installer-related uses in the repo.

Reply:
If we don't need VS.Redist.Common.NetCore.SharedFramework.x64.6.0 for anything else, I'd rather not add it. Why the installer / non-installer distinction❔

Reply:
I'll change this PR to only use the BrowserDebugHost as the non-shipping sentinel.